lib/publiccloud/azure.pm

```diff
@@ -63,8 +63,9 @@ sub find_img {
     return if (!$self->resource_exist());
 
     ($name) = $name =~ m/([^\/]+)$/;
+    my $sku = (check_var('PUBLIC_CLOUD_AZURE_SKU', 'gen2')) ? 'V2' : 'V1';
```

Comment:
I would argue that `gen2` is the default.

```suggestion
    my $sku = (check_var('PUBLIC_CLOUD_AZURE_SKU', 'gen1')) ? 'V1' : 'V2';
```

Reply:
Our workflow is currently based on gen1, it's probably a fuzz to change the default now.